pkgs/tools/security/onesixtyone/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "onesixtyone";
+  version = "unstable-2019-12-26";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "onesixtyone";
+    rev = "9ce1dcdad73d45c8694086a4f90d7713be1cbdd7";
+    sha256 = "111nxn4pcbx6p9j8cjjxv1j1s7dgf7f4dix8acsmahwbpzinzkg3";
+  };
+
+  buildPhase = ''
+    ${stdenv.cc}/bin/cc -o onesixtyone onesixtyone.c
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv onesixtyone $out/bin/onesixtyone
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Fast SNMP Scanner";
+    homepage = "https://github.com/trailofbits/onesixtyone";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Plus;
```

Since the copyright in `onesixtyone.c` gives the option of GPL2 or any later version.